client/asset/eth/eth.go

```diff
@@ -1267,8 +1292,8 @@ func (w *assetWallet) estimateSwap(lots, lotSize, feeSuggestion uint64, maxFeeRa
 		Lots:               lots,
 		Value:              value,
 		MaxFees:            maxFees,
-		RealisticWorstCase: oneGasMax * feeSuggestion,
-		RealisticBestCase:  (nSwap + allowanceGas) * feeSuggestion,
+		RealisticWorstCase: oneGasMax * rate,
+		RealisticBestCase:  (oneSwap + allowanceGas) * rate, // not even batch, just perfect match
```

Comment:
I think this is fine.

nit: I was looking at this the other day and was thinking about saying something, but isn't using nSwap what we do for other coins? In dcr the Best case is really one output isn't it? https://github.com/decred/dcrdex/blob/f9531e8709c7a80130bb73dbd36bcfbc704ccabf/client/asset/dcr/dcr.go#L1213

Reply:
And one input.  We can certainly change the UI and code docs, but: https://github.com/decred/dcrdex/blob/f9531e8709c7a80130bb73dbd36bcfbc704ccabf/client/asset/estimation.go#L20-L22
That's "best case", even better than batch.

Reply:
UI: https://github.com/decred/dcrdex/blob/f9531e8709c7a80130bb73dbd36bcfbc704ccabf/client/webserver/locales/en-us.go#L209

Consumed in single match also means single output, no need for batch.

Reply:
So is the dcr best case wrong or am I confused?

Reply:
Oh, yeah, I'm agreeing with you.  I guess we decided to consider batch because it was just there as an option, but it's hard to call it "best" when you can simply match with another order of the same number of lots.

Reply:
Yeah I think the definition of best here, as you have changed eth to, is correct.

Reply:
Yeah, seems like the `estLowFunds += dexdcr.P2SHOutputSize * (lots - 1) * bumpedNetRate ` can be deleted in **dcr** and **btc**.

Reply:
Done for dcr and btc in https://github.com/decred/dcrdex/pull/2147